sklearn/covariance/_graph_lasso.py

```diff
@@ -363,6 +393,13 @@ class GraphicalLasso(BaseGraphicalLasso):
         very sparse underlying graphs, where p > n. Elsewhere prefer cd
         which is more numerically stable.
 
+    covariance : "precomputed", default=None
```

Comment:
To be able to pass a precomputed covariance matrix in `fit`, we need to have a parameter which avoids calling `empirical_covariance`.

I think in your PR, it was the missing parameter that would make the tests fail because calling `GraphicalLasso.fit(emp_cov)` would recompute the covariance of the covariance matrix.